runtime/mangata-kusama/src/lib.rs

```diff
@@ -867,6 +866,7 @@ parameter_types! {
 	pub const TermDuration: BlockNumber = 120 * DAYS;
 	pub const DesiredMembers: u32 = 9;
 	pub const DesiredRunnersUp: u32 = 7;
+	pub const MaxVoters: u32 = 10 * 1000;
```

Comment:
```suggestion
	pub const MaxVoters: u32 = 10_000_u32;;
```

Reply:
@iStrike7 can you confirm that 10000 is proper value ?

Reply:
@mateuszaaa here https://github.com/paritytech/substrate/blob/polkadot-v0.9.28/bin/node/runtime/src/lib.rs#L969